install_config/aggregate_logging.adoc

```diff
@@ -451,7 +451,7 @@ created and ephemeral volumes are used instead.
 
 |`openshift_logging_es_pvc_prefix`
 a|Prefix for the names of persistent volume claims to be used as storage for
-Elasticsearch instances. A number is appended per instance, such as
+Elascticsearch nodes. A number is appended per node, such as
```

Comment:
Elasticsearch

Reply:
@richm Thank you and sorry. I couldn't figure out why you kept posting a fix that I made. Because the fix is  wrong! I need to find my readers...